content/systems/filecoin_vm/gas_fee.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Gas Fee
+weight: 6
+dashboardWeight: 2
+dashboardState: reliable
+dashboardAudit: missing
+dashboardTests: 0
+---
+
+# Gas Fees
+
+## Summary
+
+As is traditionally the case with many blockchains, Gas is a unit of measure of how much storage and/or compute resource an on-chain message operation consumes in order to be executed. At a high level, it works as follows: the message sender specifies the maximum amount they are willing to pay in order for their message to be executed and included in a block. This is specified both in terms of total number of units of gas (`GasLimit`), which is generally expected to be higher than the actual `GasUsed` and in terms of the price (or fee) per unit of gas (`GasFeeCap`).
```

Comment:
Maybe specify explicitely what are the different units of all the variables showed here ?